.github/workflows/test-unit-pr.yml

```
@@ -0,0 +1,106 @@
+      - name: Upload PR number
+        if: ${{ steps.parseTitle.outputs.tip }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: pr
+          path: ./pr-id.txt
+
+  pr-test:
+    if: ${{ needs.parse-components.outputs.testComponents }}
+    strategy:
+      matrix:
+        testcli: ${{ fromJson(needs.parse-components.outputs.testclis) }}
```

actionlint (1.7.4)

property "testclis" is not defined in object type {testcomponents: string}

(expression)